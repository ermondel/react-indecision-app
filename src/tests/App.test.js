import React from 'react';
import renderer from 'react-test-renderer';
import App from '../components/App';
import Modal from 'react-modal';

jest.spyOn(Modal, "setAppElement").mockImplementation(() => {});

it('<App />', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
