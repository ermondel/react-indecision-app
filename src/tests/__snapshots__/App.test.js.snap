// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="header__inner"
    >
      <h1
        className="header__title"
      >
        Indecision
      </h1>
      <h2
        className="header__subtitle"
      >
        Put your life in the hands of a computer
      </h2>
    </div>
  </header>,
  <main
    className="main"
  >
    <div
      className="action-bar"
    >
      <button
        className="main-action-btn"
        disabled={true}
        onClick={[Function]}
      >
        What should I do?
      </button>
    </div>
    <div
      className="decisions"
    >
      <div
        className="decisions__header"
      >
        <h3
          className="decisions__title"
        >
          Your Options
        </h3>
      </div>
      <p
        className="decisions__list__message"
      >
        Please add an option to get started!
      </p>
      <div
        className="decisions__add"
      >
        
        <form
          className="decisions__add__form"
          onSubmit={[Function]}
        >
          <input
            className="decisions__add__input"
            name="decision"
            type="text"
          />
          <button
            className="decisions__add__submit-btn"
            name="submitbtn"
          >
            Add Option
          </button>
        </form>
      </div>
    </div>
  </main>,
  <footer
    className="footer"
  >
    <span
      className="footer__sign"
    >
      Â©
    </span>
     
    <span
      className="footer__year"
    >
      2019 - 2021
    </span>
     
    <a
      className="footer__link"
      href="https://github.com/ermondel"
      rel="noopener noreferrer"
      target="_blank"
      title="All rights reserved | created by Serhii (ermondel)"
    >
      Serhii
    </a>
  </footer>,
]
`;
